specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachine.json

```diff
@@ -1891,6 +1891,331 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./computeRPCommon.json#/definitions/VirtualMachineApplicationsProxyResourceListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common-types/v1/common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "VirtualMachineApplications_List_MaximumSet": {
+            "$ref": "./examples/virtualMachineExamples/VirtualMachineApplications_List_MaximumSet.json"
+          },
+          "VirtualMachineApplications_List_MinimumSet": {
+            "$ref": "./examples/virtualMachineExamples/VirtualMachineApplications_List_MinimumSet.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/VMApplications/{applicationName}": {
+      "get": {
+        "tags": [
+          "VirtualMachineApplications"
+        ],
+        "operationId": "VirtualMachineApplications_Get",
+        "description": "The operation to get the status of an application on a Virtual Machine.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "vmName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual machine."
+          },
+          {
+            "name": "applicationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the application to query."
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation.",
+            "enum": [
+              "instanceView"
+            ],
+            "x-ms-enum": {
+              "name": "ApplicationInstanceView",
+              "modelAsString": false
+            }
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./computeRPCommon.json#/definitions/VMApplicationProxyResourceWithInstanceView"
```

Comment:
>VMApplicationProxyResourceWithInstanceView [](http://example.com/codeflow?start=59&length=42)

make the names user friendly as these go into the SDKs

VirtualMachineApplication ?

Reply:
VMApplications is the product name, renamed the object to VMApplicationWithInstanceView

Reply:
sorry camel case vmApplicationWithInstanceView